scala3doc/src/dotty/dokka/model/api/api.scala

```diff
@@ -146,49 +140,38 @@ object HierarchyGraph:
   def withEdges(edges: Seq[(LinkToType, LinkToType)]) = HierarchyGraph.empty ++ edges
 
 
-type Member = Documentable
+case class Member(
+  name: String,
+  dri: DRI,
+  kind: Kind,
+  visibility: Visibility = Visibility.Unrestricted,
+  modifiers: Seq[dotty.dokka.model.api.Modifier] = Nil,
+  annotations: List[Annotation] = Nil,
+  signature: Signature = Signature(),
+  sources: Option[TastyDocumentableSource] = None,
+  origin: Origin = Origin.RegularlyDefined,
+  inheritedFrom: Option[InheritedFrom] = None,
+  graph: HierarchyGraph = HierarchyGraph.empty,
+  docs: Option[Comment] = None,
+  members : Seq[Member] = Nil,
+  directParents: Seq[LinkToType] = Nil,
+  parents: Seq[LinkToType] = Nil,
+  knownChildren: Seq[LinkToType] = Nil,
+  companion: Option[DRI] = None,
+)
 
 object Member:
-  def unapply(d: Documentable): Option[(String, DRI, Visibility, Kind, Origin)] =
-    d.memberExt.map(v => (d.getName, d.getDri, v.visibility, v.kind, v.origin))
+  def unapply(v: Member): Option[(String, DRI, Visibility, Kind, Origin)] =
+    Some((v.name, v.dri, v.visibility, v.kind, v.origin))
 
 extension[T] (member: Member)
+  def asLink: LinkToType = LinkToType(member.signature, member.dri, member.kind)
+  def deprecated: Option[Annotation] =
```

Comment:
This will probably conflict with that PR https://github.com/lampepfl/dotty/pull/11216 Though I can resolve conflicts on my own

Reply:
Note to myself, not CR